g-vega-cl/game-of-life
# Build Conway's Game of Life

# Description
Build Conway's game of life. The steps followed were:
1. Build Grid
2. Create simulate function
3. Use useContext to pass props between react siblings


# QA Steps
1. Run the repo
4. Click the "Start" button and make sure it now says "Pause"
5. Click the clear button and make sure the grid is cleared.
6. Click the randomize button and make sure the grid is randomized. Notice that randomize does not pause or start the game, the game keeps running as it was.
7. Clear button pauses the game.
8. Start button creates a random grid if the grid is empty and then starts the game
